Tests/UnitTests/Sources/GREYTestHelperTest.m

```diff
@@ -0,0 +1,39 @@
+//
+// Copyright 2016 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#import "GREYBaseTest.h"
+
+#import "Provider/GREYUIWindowProvider.h"
+
+@interface GREYTestHelper_Test : GREYBaseTest
+
+@end
+
+@implementation GREYTestHelper_Test
+
+- (void)testAnimationSpeed {
+  [GREYTestHelper increaseAnimationSpeed];
+  for (UIWindow *window in [GREYUIWindowProvider allWindows]) {
+    XCTAssertEqual([[window layer] speed], 100, @"Animation speed not 100");
```

Comment:
Instead of checking for an exact value, check for increase so you dont have to hardcode 100 and keep maintaining it.